kivy/tests/test_window_opacity.py

```diff
@@ -0,0 +1,25 @@
+from kivy.tests.common import GraphicUnitTest
+
+
+class WindowOpacityTest(GraphicUnitTest):
+
+    def test_window_opacity_property(self):
+        from kivy.core.window import Window
+        
+        opacity = 0.5
+        Window.opacity = opacity
+        self.assertEqual(Window.opacity, opacity)
+
+        from kivy.logger import LoggerHistory
+        
+        Window.opacity = 15
+        self.assertEqual(LoggerHistory.history[0].msg, 
+                         'Window: The opacity value of '
+                         'the window should be in the '
+                         'range from 0.0 to 1.0.')
+        
+        LoggerHistory.clear_history()
+        
+        Window.opacity = -15
+        assert len(LoggerHistory.history) == 1
+
```

Comment:
Hi!
It likely makes sense to have this test into `test_window_base.py` instead of using a specific file.
What do you think about it?

Reply:
Ok, no problems

Reply:
Done